src/cpu/meson.build

```diff
@@ -40,11 +40,18 @@ endforeach
 if conf_data.has('C_DYNAMIC_X86')
   message('Building dynamic core optimized for x86/x86_64 (dyn-x86) ' +
           'for ' + conf_data.get('C_TARGETCPU'))
+  summary('Dynamic core',
+          'dyn-x86 (optimized for x86/x86_64)',
+          section : 'Emulator Features')
 elif conf_data.has('C_DYNREC')
   message('Building dynamic core (dynrec) ' +
           'for ' + conf_data.get('C_TARGETCPU'))
```

Comment:
Since we're touching these adjacent lines, we can slightly normalize the `messages`

```
  message('Building x86/x86_64-optimized dynamic core (dyn-x86) for ' +
          conf_data.get('C_TARGETCPU'))
```

```
message('Building generic dynamic core (dynrec) for ' +
          conf_data.get('C_TARGETCPU'))
```